### PR TITLE
Make pylint and pycodestyle fixes

### PR DESCRIPTION
**Pull request by matthewrsj**
Make Ister pylint and pycodestyle clean. Many exceptions are removed, and those that remain are renamed to use their title instead of error number.

Signed-off-by: Matthew Johnson <matthew.johnson@intel.com>

### CODE REVIEW

File: ister_gui.py [hunk 1]
```diff
@@ -403,8 +420,8 @@ def construct(self):
                                  height=('relative', PERCENTAGE_H))
         self._ui = urwid.AttrMap(self._ui, 'banner')
 
-    # This callback is registered with urwid.MainLoop and gives us the
-    # opportnity to intercept keystrokes and handle things like tabs.
+    # This callback is registered with urwid.MainLoop and gies us the
+    # opportnity to intercept keystrokes and handle things lke tabs.
```
Comment (bryteise) [inline ister_gui.py]: Introduced two typos here.

**Comment by bryteise** [comment]
Nice cleanup. +1 aside from the typos.

File: ister_gui.py (hunk 1 above)
Comment (phmccarty) [inline ister_gui.py]: you could also s/opportnity/opportunity as well :)

**Comment by matthewrsj** [comment]
@bryteise typos fixed

**Comment by matthewrsj** [comment]
@bryteise everything is how it should be. ister_test.py cleaned up as well.

**Pull request merged**